recipes/hide_code/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "hide_code" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "009fdd3f3f150db18037534f549d2a926ab43832260955c5447b28e7414fbf53" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/h/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    noarch: python
+    skip: True  # [win or osx]
+    script: |
+        set -ex
+        python setup.py install --single-version-externally-managed --record record.txt
+        jupyter nbextension install --py hide_code --sys-prefix
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - notebook >=4.4.1
+        - python-pdfkit
```

Comment:
Do we need `python-pdfkit` at build time?

Reply:
Yes this is needed because of the `jupyter nbextension install` line above, which imports the module.